src/constants/shaders/slice.frag

```diff
@@ -87,7 +86,6 @@ void main() {
   vec4 pos = vec4(vUv, 
     (SLICES==1.0 && Z_SLICE==0) ? 0.0 : float(Z_SLICE) / (SLICES - 1.0), 
     0.0);
-  pos.xyz = (pos.xyz - SUBSET_OFFSET) / SUBSET_SCALE;
```

Comment:
removing this line is the bug fix.  For example: when pos.z = 1.0, we could get into a situation where this math was doing (1.0 - 0.9666) /  0.0333 which is slightly greater than 1.  Then later there is bounds checking code which needs to ensure that pos is never greater than 1.